1.10.7/alpine3.10/include/pip-constraints.txt

```diff
@@ -13,3 +13,6 @@ WTforms<2.3.0
 
 # Details: https://github.com/apache/airflow/issues/8599
 flask-appbuilder<2.3.3
+
+# Details: https://github.com/apache/airflow/pull/8833
+azure-storage<0.37.0
```

Comment:
Didn't we have all these constaints in a previous PR?

Reply:
Yeah ignore the constraints for now, I just rebased over that PR as the tests were failing